pyiron_contrib/workflow/node.py

```diff
@@ -277,9 +277,18 @@ def server(self, server: Server | None):
         self._server = server
 
     def disconnect(self):
-        self.inputs.disconnect()
-        self.outputs.disconnect()
-        self.signals.disconnect()
+        """
+        Disconnect all connections belonging to inputs, outputs, and signals channels.
+
+        Returns:
+            [list[tuple[Channel, Channel]]]: A list of the pairs of channels that no
+                longer participate in a connection.
+        """
+        destroyed_connections = []
+        destroyed_connections.extend(self.inputs.disconnect())
+        destroyed_connections.extend(self.outputs.disconnect())
+        destroyed_connections.extend(self.signals.disconnect())
+        return destroyed_connections
```

Comment:
```suggestion
        return self.inputs.disconnect() + self.outputs.disconnect() + self.signals.disconnect()
```

If the line is not too long